File: mocks/psak-mock/src/main/java/no/nav/util/PenNAVEnhetUtil.java
package no.nav.util;

import no.nav.lib.pen.psakpselv.asbo.navorgenhet.ASBOPenNAVEnhet;

public class PenNAVEnhetUtil {

    public static ASBOPenNAVEnhet getAsboPenNAVEnhet() {
        ASBOPenNAVEnhet enhet = new ASBOPenNAVEnhet();
        enhet.setEnhetsId("4407");
        enhet.setEnhetsNavn("NAV Arbeid og ytelser Tønsberg");
        enhet.setOrgNivaKode("GR");
        enhet.setOrgEnhetsId("7000");
        return enhet;
    }

}

File: vtp-pensjon-application/src/main/kotlin/no/nav/pensjon/vtp/mocks/psak/util/PenNAVEnhetUtil.kt
package no.nav.pensjon.vtp.mocks.psak.util

import no.nav.lib.pen.psakpselv.asbo.navorgenhet.ASBOPenNAVEnhet

fun asboPenNAVEnhet(): ASBOPenNAVEnhet {
    return ASBOPenNAVEnhet().apply {
        enhetsId = "4407"
        enhetsNavn = "NAV Arbeid og ytelser Tønsberg"
        orgNivaKode = "GR"
        orgEnhetsId = "7000"
    }
}
